test/unit/queue/src/QueueTest.cpp

```diff
@@ -278,3 +280,21 @@ TEMPLATE_LIST_TEST_CASE("isQueue", "[queue]", alpaka::test::TestQueues)
 
     REQUIRE(alpaka::isQueue<decltype(f.m_queue)>);
 }
+
+#if defined(ALPAKA_ACC_GPU_CUDA_ENABLED) || defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+
+#    if defined(ALPAKA_ACC_GPU_CUDA_ENABLED)
+using Api = alpaka::ApiCudaRt;
+#    elif defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+using Api = alpaka::ApiHipRt;
+#    endif
+
+TEMPLATE_LIST_TEST_CASE("constructQueueWithStream", "[queue]", alpaka::test::TestQueues)
+{
+    Api::Stream_t stream;
+    ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(Api::streamCreate(&stream));
+    auto queue = alpaka::QueueUniformCudaHipRtBlocking<Api>(stream);
+    ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK_NOEXCEPT(Api::streamDestroy(stream));
```

Comment:
```suggestion
    ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(Api::streamDestroy(stream));
```
?

I think we only use `ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK_NOEXCEPT` in destructors.